Resources/Textures/Structures/Windows/cracks_directional.rsi/meta.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/tgstation/tgstation at commit e06b82a7f4b2b09216fb28fd384c95a2e1dc50e5 and modified by Darkie",
```

Comment:
Please don't copy-paste copyright, use the commit you actually got it from as this is 2 years old

Reply:
Fixed! I think. 
This is the commit I got them from: https://github.com/vulppine/space-station-14/commit/f57e8ec6b9b4b72ef56c8146be0bc159ed2691ee

Reply:
Did flipp make these sprites or did they get it from somewhere?

Reply:
I'm not sure, I can't find the original source from the original copyright. I took the cracks sprites that were already there for regular windows and cropped them in order to make them work. Maybe the original copyright for the crack sprite is wrong to begin with.

Reply:
Ah, I think I found the original source: [e34888373a9436af3f09672c2d92757721b5a94c](https://github.com/vulppine/space-station-14/commit/e34888373a9436af3f09672c2d92757721b5a94c) . This is when the crack files seem to have been created.

Reply:
Looking at that commit really lags out my browser, it's massive.

Reply:
And digging deeper, looks like originally made here https://github.com/SweptWasTaken/space-station-14/commit/d59cc139b275fde1e6c1328c7862aab98e9cb2f6 by Vera.

Reply:
And that's where the taken from TGSstation copyright is also from.
So... what do?